torch_xla/csrc/runtime/pjrt_computation_client.cc

```diff
@@ -559,6 +595,13 @@ std::vector<ComputationClient::ComputationPtr> PjRtComputationClient::Compile(
   return computations;
 }
 
+torch::lazy::hash_t PjRtComputationClient::HashCompilationEnv() {
+  // TODO(jonbolin): Incorporate CompileOptions into the hash. These are
```

Comment:
Once we do, we really don't need the SPMD mode be part of the hash (virtual device on/off).